src/components/a-checkbox/_preview.html

```diff
@@ -1,2 +1,5 @@
-<axa-checkbox id="test" value="a" name="test-name" ></axa-checkbox>
```

Comment:
This has visible layout issues that need to be resolved before merge:

<img width="186" alt="bildschirmfoto 2018-02-16 um 09 18 27" src="https://user-images.githubusercontent.com/275235/36298850-72471d96-12fa-11e8-8d74-5ccc7e1b4d64.png">